src/UI/Art/SpriteArtist/SpriteInfoView.java

```diff
@@ -0,0 +1,142 @@
+    int paneVSize = (int) (viewHeight   ) - paneOuterBuffer*4; // height ""
+
+    // Get the current menu selections.
+    Commander thisCO = myControl.getSelectedCO();
+    InfoPage page = myControl.getSelectedPage();
+
+    // Draw the commander art.
+    BufferedImage COPic = SpriteLibrary.getCommanderSprites(myControl.getSelectedCOInfo().name).body;
+    // justify bottom/right
+    g.drawImage(COPic, viewWidth - COPic.getWidth()*drawScale, viewHeight - COPic.getHeight()*drawScale,
+        COPic.getWidth()*drawScale, COPic.getHeight()*drawScale, null);
+
+    if (null != thisCO)
+      CommanderOverlayArtist.drawCommanderOverlay(g, thisCO, false);
+    
+    // add the actual info
+    g.setColor(SpriteUIUtils.MENUFRAMECOLOR); // outer buffer
+    g.fillRect(  paneOuterBuffer,   paneOuterBuffer, paneHSize + 2*paneOuterBuffer, paneVSize + 2*paneOuterBuffer);
+    g.setColor(SpriteUIUtils.MENUBGCOLOR);    // inside of the pane
+    g.fillRect(2*paneOuterBuffer, 2*paneOuterBuffer, paneHSize                    , paneVSize                    );
+    
+    // TODO: consider drawing all pages as one big image, so the user can scroll smoothly through it regardless of screen size
+    int drawingWidth = paneHSize - paneOuterBuffer;
+    switch (page.pageType)
+    {
+      case CO_HEADERS:
+        int overlayHeight = 42*drawScale;
+        int heightOffset = 0;
+        for (Commander CO : myControl.getGame().commanders)
+        {
+          // Build our overlay
+          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(drawingWidth, overlayHeight);
+          CommanderOverlayArtist.drawCommanderOverlay(overlayPic.getGraphics(), CO, true);
+
+          // Add brief status text per CO
+          String status = new COStateInfo(getDrawableMap(myControl.getGame()), CO).getAbbrevStatus();
+          BufferedImage statusText = SpriteUIUtils.paintTextNormalized(status, drawingWidth);
+          overlayPic.getGraphics().drawImage(statusText, 0, drawScale * (5+SpriteLibrary.getCoOverlay(CO, true).getHeight()), null);
+
+          // Drop the overlay where it's supposed to go
+          g.drawImage(overlayPic,2*paneOuterBuffer, 2*paneOuterBuffer + heightOffset, null);
+          heightOffset += overlayHeight;
+        }
+        break;
+      case GAME_STATUS:
+        if( null != thisCO )
+        {
+          String status = new COStateInfo(getDrawableMap(myControl.getGame()), thisCO).getFullStatus();
+          BufferedImage statusText = SpriteUIUtils.paintTextNormalized(status, drawingWidth);
+          g.drawImage(statusText, 3 * paneOuterBuffer, 3 * paneOuterBuffer, null);
+        }
+        break;
+      case BASIC:
+        BufferedImage infoText = SpriteUIUtils.paintTextNormalized(page.info, drawingWidth);
+        g.drawImage(infoText,3*paneOuterBuffer, 3*paneOuterBuffer, null);
+        break;
+    }
+  }
+
+  
+  // Vestigial method stubs from MapView
```

Comment:
Vestigial stubs make me uneasy.

Reply:
They aren't used, and MapView is where the viewable map finding logic lives.

I guess I could move that to UIUtils?

Reply:
ehhh, I'd just leave it for now. MapView seems like a logical place to have map-viewing logic. I'll let the uneasiness fester until it either goes away or causes actionable heartburn.